javascript/javascript1/week2/candy_helper.js

```diff
@@ -0,0 +1,49 @@
+let boughtCandyPrices = [];
+function addCandy(candyType, weight) {
+    let pricePerGram = 0;
+    switch (candyType) {
+        case 'Sweet':
+            pricePerGram = 0.5;
+            break;
+        case 'Chocolote':
+            pricePerGram = 0.7;
+            break;
+        case 'Toffee':
+            pricePerGram = 1.1;
+            break;
+        case 'Chewing-gum':
+            pricePerGram = 0.03;
+            break;
+        default:
+            console.log('Wrong type, select something else');
+    }
+    let price = pricePerGram * weight;
+    boughtCandyPrices.push(price);
+}
+
+let amountToSpend = Math.random() * 100;
+let boughtCandy = [
+    ['Sweet', 10],
+    ['Chocolote', 20],
+    ['Toffee', 15],
+    ['Chewing-gum', 100],
+];
+
+function canBuyCandy() {
+    for (let i=0; i < boughtCandy.length; i++) {
+        let candy = boughtCandy[i];
+        addCandy(candy[0], candy[1]);
```

Comment:
what if you write addCandy(boughtCandy[i][0], boughtCandy[i][1]);

Reply:
fixed